app/src/main/java/org/wikipedia/language/FirebaseLanguageDetector.kt

```diff
@@ -0,0 +1,20 @@
+package org.wikipedia.language
+
+import com.google.mlkit.nl.languageid.LanguageIdentification
+
+class FirebaseLanguageDetector {
```

Comment:
Just noting down what we discussed earlier today:
After the `push` PR is merged, you would move this class to the `extra` source tree, and then create an identical class in the `fdroid` source tree, except with all the functions being blank.

Reply:
this is now done @dbrant . I have added the necessary implementations in gradle, and created the class in `extra` and a stubbed it on fdroid source tree.